### PR TITLE
add a test for highscores integrity of the list

### PR DESCRIPTION
**Pull request by valentin-p**
I want to help this PR/issue #1352 so I implemented the test for highscores in the generator file

Because of the PR #1678 the version number disappeared on the test file

### CODE REVIEW

File: generators/Exercises/Generators/HighScores.cs [hunk 1]
```diff
@@ -11,12 +11,26 @@ protected override void UpdateTestMethod(TestMethod testMethod)
...
+        protected override void UpdateTestClass(TestClass testClass)
+            => AddTestMethodForPersonalBest(testClass);
+
+        protected override void UpdateNamespaces(ISet<string> namespaces)
             => namespaces.Add(typeof(List<int>).Namespace);
+
+        private static void AddTestMethodForPersonalBest(TestClass testClass)
+        {
+            testClass.AdditionalMethods.Add(@"[Fact(Skip = ""Remove this Skip property to run this test"")]
+public void Lastest_score_should_not_change_after_calling_personal_best()
```
Comment (ErikSchierboom) [inline generators/Exercises/Generators/HighScores.cs:28]: Small typo: 

```suggestion
public void Latest_score_should_not_change_after_calling_personal_best()
```

File: exercises/high-scores/HighScoresTests.cs [hunk 2]
```diff
@@ -60,4 +60,12 @@ public void Personal_top_when_there_is_only_one()
         var sut = new HighScores(new List<int> { 40 });
         Assert.Equal(new List<int> { 40 }, sut.PersonalTopThree());
     }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void Lastest_score_should_not_change_after_calling_personal_best()
```
Comment (ErikSchierboom) [inline exercises/high-scores/HighScoresTests.cs:65]:
```suggestion
    public void Latest_score_should_not_change_after_calling_personal_best()
```

**Review by ErikSchierboom — COMMENTED** [review]
Awesome! Found one tiny typo.

File: generators/Exercises/Generators/HighScores.cs (hunk 1 above)
Comment (valentin-p) [reply generators/Exercises/Generators/HighScores.cs]: funny one, it was in the original PR too.

**Comment by valentin-p** [comment]
Should be good to go

**Comment by valentin-p** [comment]
A total of 1 test files matched the specified pattern.
[xUnit.net 00:00:00.81]     DndCharacterTests.Random_ability_is_distributed_correctly [FAIL]
  X DndCharacterTests.Random_ability_is_distributed_correctly [161ms]
  Error Message:
   Assert.InRange() Failure
Range:  (80 - 120)
Actual: 72
  Stack Trace:
     at DndCharacterTests.Random_ability_is_distributed_correctly() in /home/runner/work/csharp/csharp/build/dnd-character/DndCharacterTests.cs:line 172
Test Run Failed.

Total tests: 20
     Passed: 19
     Failed: 1
 Total time: 2.2381 Seconds

**Comment by valentin-p** [comment]
Should be good to go now, the previous error was temporary.

**Comment by ErikSchierboom** [comment]
> Should be good to go now, the previous error was temporary.

Ah yes, the joys of handle tests deal with randomness :)

**Review by ErikSchierboom — APPROVED** [review]
Excellent work!

**Comment by ErikSchierboom** [comment]
Closes #1422

**Pull request merged**